packages/tql/src/dialects/postgres.ts

```diff
@@ -0,0 +1,104 @@
+import { IdenticalColumnValidator } from '../values-object-validator';
+import type { DialectImpl } from '../types';
+import { BaseDialect } from '../dialect';
+import {
+  type TqlIdentifiers,
+  type TqlList,
+  type TqlParameter,
+  type TqlTemplateString,
+  type TqlSet,
+  type TqlValues,
+} from '../nodes.js';
+
+export class PostgresDialect extends BaseDialect implements DialectImpl {
+  string(str: TqlTemplateString): void {
+    this.appendToQuery(str.value);
+  }
```

Comment:
Are we wrapping this value in quotes in the query? If not, is there a reason we call this `string`? It feels like `raw` or something along those lines might be a more apt name.

Reply:
It's called a `TqlTemplateString` (see the JSDoc on that class) -- though I wouldn't be against renaming it! TemplateStrings are only created by the develper-typed portions of `query` and `fragment` objects -- i.e. in:

```ts
const q = query`SELECT * FROM user WHERE user_id = ${1234} AND LOWER(user_name) LIKE 'jingleheimer%'`
```

You'd get this AST:

```ts
const ast = new TqlQuery([
  new TqlTemplateString('SELECT * FROM user WHERE user_id = '),
  new TqlParameter(1234),
  new TqlTemplateString(" AND LOWER(user_name) LIKE 'jingleheimer%'"),
])
```

Since we prevent `query` and `fragment` from being called in any way other than a template expression, we know these are always safe to append to the query directly, as they only could have been written by the developer. (The one exception is `unsafe`, which creates a `TqlTemplateString` directly, but... well, it's in the name. It's unsafe.)